projects/bp-gallery/cypress/e2e/archives.cy.ts

```diff
@@ -79,8 +68,63 @@ describe('Archives View', () => {
     cy.contains('Test-Link 1 Edit');
     cy.contains('Test-Link 3');
   });
-  it('successfully sets an image as showcase picture when pressing on the star button', () => {
+
+  it('successfully sets an image as showcase picture when pressing on the star button via show more for keywords', () => {
+    cy.get('.overview-container:eq(1)').contains('Mehr anzeigen').click();
+    cy.get(`[data-testid="StarIcon"]`).eq(1).click();
+    cy.contains('Zurück').click();
+    cy.get('.archive-showcase').children().should('contain.id', 'picture-preview-for-4');
```

Comment:
I gonna change the things I agree with for tests modified in this branch, because waiting for common guidelines, would probably cause more pain, than doing some things twice.